mozilla-release/browser/themes/shared/urlbar-searchbar.inc.css

```diff
@@ -261,7 +261,7 @@
 
 .urlbar-icon {
   width: 28px;
-  height: 30px;
+  height: 28px;
```

Comment:
I took a look at what FF had here.
It has 28px so I rolled that change (I made myself earlier) back.
So please do not be surprised of why I provided that change here.